test1-dev.js

```diff
@@ -3,3 +3,10 @@
 if(foo  === "bar") {
   console.log( 'hey' )
 }
+
+
+var anotherAnanas = '' ;
+
+if(anotherAnanas  === "aaaabar") {
+  console.log( 'aaaaaahey' )
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://release.k8s.dev.codacy.org/gh/machadoit-test-org/eslint-test-dev/pullRequest?prid=1983)
```suggestion
  console.log( "aaaaaahey" )
```